api/documents/views.py

```diff
@@ -0,0 +1,37 @@
+from http import HTTPStatus
+from django.http import HttpRequest
+
+from documents.models import Document
+from documents.schemas import DocumentIn, DocumentOut
+from documents.controllers import *
+
+from ninja import Router
+from ninja.pagination import LimitOffsetPagination, paginate
+
+document_router = Router(tags=["Documents"])
```

Comment:
It's obvious it's document router
```suggestion
router = Router(tags=["Documents"])
```
(Edit other places accordingly)